ci/Jenkinsfiles/build.groovy

```diff
@@ -119,6 +160,172 @@ pipeline {
         }
       }
     }
+    stage('Build Docker Image') {
+      when {
+        anyOf {
+          branch 'PR-*'
+          branch "${REFERENCE_BRANCH}"
+        }
+      }
+      steps {
+        setGitHubBuildStatus('docker/build', 'Build Docker Image', 'PENDING')
+        container('maven') {
+          echo """
+          ------------------------------------------
+          Build Common Sample Docker Image
+          ------------------------------------------
+          Image tag: ${VERSION}
+          Registry: ${DOCKER_REGISTRY}
+          Nuxeo Image tag: ${NUXEO_IMAGE_VERSION}
+          """
+          script {
+            // build and push Docker images to the Jenkins X internal Docker registry
+            def dockerPath = 'docker'
+            sh "envsubst < ${dockerPath}/skaffold.yaml > ${dockerPath}/skaffold.yaml~gen"
+            withCredentials([string(credentialsId: 'instance-clid', variable: 'INSTANCE_CLID')]) {
+
+              // replace lines by "--"
+              String clid = sh(returnStdout: true, script: '''#!/bin/bash +x
+              echo -e "${INSTANCE_CLID}" | sed ':a;N;\$!ba;s/\\n/--/g'
```

Comment:
Wow :)

Reply:
Yeah well, we do have this in all pipelines (carriage return in the clid is not handled well... so it's replaced).

Thanks for the review: Antoine already had a look at the docker/Jenkins part so we should be good :)